crates/mun_runtime/tests/arrays.rs

```diff
@@ -0,0 +1,116 @@
+use itertools::Itertools;
+use mun_runtime::{ArrayRef, StructRef};
+use mun_test::CompileAndRunTestDriver;
+
+#[test]
+fn arrays() {
+    let driver = CompileAndRunTestDriver::new(
+        r"
+    pub fn main() -> [i32] { [5,4,3,2,1] }
+    ",
+        |builder| builder,
+    )
+    .expect("Failed to build test driver");
+
+    let result: ArrayRef<'_, i32> = driver.runtime.invoke("main", ()).unwrap();
+
+    assert_eq!(result.len(), 5);
+    assert!(result.capacity() >= 5);
```

Comment:
Why not equal to 5?

Reply:
Because in the future we might allocate a little more memory to accommodate extra pushing. The requirement on the capacity is that it can at least store `5` elements or more.